grafana-plugin/src/components/IntegrationCollapsibleTreeView/IntegrationCollapsibleTreeView.tsx

```diff
@@ -103,19 +111,21 @@ const IntegrationCollapsibleTreeView: React.FC<IntegrationCollapsibleTreeViewPro
 
 const IntegrationCollapsibleTreeItem: React.FC<{
   item: IntegrationCollapsibleItem;
+  elementPosition?: number;
   isExpanded: boolean;
   onClick: () => void;
-}> = ({ item, isExpanded, onClick }) => {
-  const iconOnClickFn = !item.isCollapsible ? undefined : onClick;
+}> = ({ item, elementPosition, isExpanded, onClick }) => {
+  const handleIconClick = !item.isCollapsible ? undefined : onClick;
```

Comment:
more straightforward and easy to read:
`const handleIconClick = item.isCollapsible ? onClick : undefined;`